server/views/pages/about.njk

```diff
@@ -46,6 +46,10 @@
 
 {% block content %}
     {{ super() }}
```

Comment:
I know this isn't from the PR, but any idea why this is here?

Reply:
I'll pick this up with you on slack.